pages/common/poetry.md

```diff
@@ -0,0 +1,24 @@
+# poetry
+
+> Manage Python packages and dependencies.
+> More information: <https://python-poetry.org/docs>.
+
+- Create a new Poetry project in the directory with a specific name:
+
+`poetry new {{project_name}}`
+
+- Install a dependency and its subdependencies:
+
+`poetry add {{dependency}}`
+
+- Interactively initialize the current directory as a new Poetry project:
+
+`poetry init`
+
+- Get the latest version of all dependencies and update poetry.lock:
```

Comment:
poetry.lock inside backticks(``)

Reply:
Is this an official project's style? There is no mention of it in the style guide.

Reply:
I thought we had decided against it at one point. But I can't remember.

Reply:
It seems there has been no final decision there. Should a new discussion be opened?

Reply:
These decisions are always tricky :P But from this https://github.com/tldr-pages/tldr/pull/3114#issuecomment-506065555, it seems like putting poetry.lock inside backticks was an unambiguous decision and nobody was particularly against it.